docs/tutorial/tutorial-3.rst

```diff
@@ -283,17 +283,18 @@ You can now use Briefcase to run your application:
 
       (beeware-venv) C:\...>
 
-This will start your run your native application, using the output of the
+This will start to run your native application, using the output of the
 `build` command.
 
-You may notice some small differences in the way your application looks when
-it's running - for example, icons, and the name displayed by the operating
-system, may be slightly different to those you saw when running under developer
-mode. This is because you're using the actual packaged application, not just
+You'll notice that the console output we saw earlier won't work anymore. This is because we are now running a standalone, packaged app that has no console to output to.
```

Comment:
It might be better to say "won't be visible" rather than "won't work".. It's still *working* - it's just not in a place you can see it (because standard in/out have been redirected to the console log).

Also, a gramatical nitpick - ending sentences with a proposition: "...has no (visible) console to which it can output", rather than "... has no console to output to.".